From f92f6fe545f2809418752e72b22439d614f9d429 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 25 May 2017 00:16:14 +0800
Subject: [PATCH 1815/5242] MLK-14975-1 ARM64: dts: freescale: imx8qm:
 increase DDR space for M4

commit  39334f804726d029da8086ebb63ae981369869e0 from
https://source.codeaurora.org/external/imx/linux-imx.git

M4 has use case of video and need more DDR space
for video buffer, reserve more DDR space for it.

0x80000000 - 0x800fffff: ARM Trusted Firmware;
0x80100000 - 0x87ffffff: M4/RPMSG;

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index a8c0c8e..0eb1f71 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -16,8 +16,8 @@
 /dts-v1/;
 
 /* First 128KB is for PSCI ATF. */
-/* Last 3M is for M4/RPMSG */
-/memreserve/ 0x80000000 0x00400000;
+/* Last 127M is for M4/RPMSG */
+/memreserve/ 0x80000000 0x08000000;
 
 #include "fsl-imx8qm.dtsi"
 
-- 
1.7.9.5

